src/clj_sqs_extended/serdes.clj

```diff
@@ -63,22 +62,23 @@
         reader (transit/reader in :json read-handlers)]
     (transit/read reader)))
 
+(defn- unsupported-format-exception
+  [got]
+  (ex-info (format "Only %s formats are supported. We received %s."
+                   [:transit :json] got) {}))
```

Comment:
new line with `{}` since it's the 2nd arg

also, perhaps pass `{:cause :unsupported-serde-format}` instead of empty map

Reply:
@daisybytes you can merge after this change